scripts/seed_datastore.py

```diff
@@ -0,0 +1,147 @@
+        ]
+        logging.info('Adding %d recent features of %d total to the datastore.',
+                     len(recent_features), len(parsed_features))
+
+        for f in recent_features:
+            fe = FeatureEntry(
+                id=f['id'],
+                created=datetime.fromisoformat(f['created']['when']),
+                updated=datetime.fromisoformat(f['updated']['when']))
+            fe.name = f['name']
+            fe.summary = f['summary']
+            fe.category = f.get('category_int', MISC)
+            # fe.feature_type = f['feature_type']  # Not available from the dump?
+            fe.impl_status_chrome = f['browsers']['chrome']['status']['val']
+            fe.standard_maturity = f['standards']['maturity']['val']
+            fe.ff_views = f['browsers']['ff']['view']['val']
+            fe.safari_views = f['browsers']['safari']['view']['val']
+            fe.web_dev_views = f['browsers']['webdev']['view']['val']
+
+            x = ({
+                'unlisted': fe.unlisted,
+                'breaking_change': fe.breaking_change,
+                'blink_components': fe.blink_components or [],
+                'resources': {
+                    'samples': fe.sample_links or [],
+                    'docs': fe.doc_links or [],
+                },
+                'created': {
+                    'by': fe.creator_email,
+                },
+                'updated': {
+                    'by': fe.updater_email,
+                },
+                'standards': {
+                    'spec': fe.spec_link,
+                },
+                'browsers': {
+                    'chrome': {
+                        'bug': fe.bug_url,
+                        'blink_components': fe.blink_components or [],
+                        'devrel': fe.devrel_emails or [],
+                        'owners': fe.owner_emails or [],
+                        'prefixed': fe.prefixed,
+                    },
+                    'ff': {
+                        'view': {
+                            'url': fe.ff_views_link,
+                            'notes': fe.ff_views_notes,
+                        }
+                    },
+                    'safari': {
+                        'view': {
+                            'url': fe.safari_views_link,
+                            'notes': fe.safari_views_notes,
+                        }
+                    },
+                    'webdev': {
+                        'view': {
+                            'url': fe.web_dev_views_link,
+                            'notes': fe.web_dev_views_notes,
+                        }
+                    },
+                    'other': {
+                        'view': {
+                            'notes': fe.other_views_notes,
+                        }
+                    },
+                }
+            })
+            fe.put()
+
+    if args.admin:
+        logging.info('Making %s an admin.', args.admin)
```

Comment:
Could you move this into a function too? Example `def add_admin`

Reply:
Done.